test/integration/monitoring-alert/monitoring_alert_test.go

```diff
@@ -0,0 +1,42 @@
+package monitoring_alert
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/gcloud"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/tft"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMonitoringAlertSecret(t *testing.T) {
+	secretT := tft.NewTFBlueprintTest(t)
+
+	secretT.DefineVerify(func(assert *assert.Assertions) {
+		secretT.DefaultVerify(assert)
+
+		projectDescribe := gcloud.Runf(t, "projects describe %s", secretT.GetStringOutput("project_id"))
+		projectNUM := projectDescribe.Get("projectNumber").String()
+		secretDescribe := gcloud.Runf(t, "secrets describe %s --project %s", "secret-1", secretT.GetStringOutput("project_id"))
+		secretName := secretDescribe.Get("name").String()
+		assert.Equal(fmt.Sprintf("projects/%s/secrets/secret-1", projectNUM), secretName, "has expected name")
+
+		monitoringChannel := gcloud.Runf(t, "beta monitoring channels list --project %s --filter name='%s' --format=json", secretT.GetStringOutput("project_id"), secretT.GetStringOutput("notification_channel_name")).Array()
```

Comment:
is it necessary to add `--format=json` to the gcloud command? this should be automatic when using  `gcloud.Runf`